test/gtest/miopendriver_regression_big_tensor.cpp

```diff
@@ -70,10 +68,15 @@ bool IsTestSupportedForDevice()
 
 void RunMIOpenDriver()
 {
-    bool runTestSuite = miopen::IsEnabled(MIOPEN_ENV(MIOPEN_TEST_WITH_MIOPENDRIVER)) &&
-                        IsTestSupportedForDevice() &&
-                        miopen::IsEnabled(MIOPEN_ENV(MIOPEN_TEST_ALL)) &&
-                        miopen::GetStringEnv(MIOPEN_ENV(MIOPEN_TEST_FLOAT_ARG)) == "--float";
+    // For determining if we should run test suite, ensure that test is supported on the hardware.
```

Comment:
A logical follow up step would be to pick these and the `MIOPEN_DECLARE_ENV...` calls into a common header file and then you have to write the comment only once :).

Reply:
Agreed! I believe we made a separate task for doing that in bulk across the tests, but could take the first step now if it makes sense. Thoughts?

Reply:
I say start now and then you or someone else can finish it up in a follow up PR. That's my personal preference.

Reply:
@amberhassaan Updated, can you check to see if you like the changes? 

Thanks!

Reply:
Yes, this is great. Thank you.